is_core/generic_views/inlines/__init__.py

```diff
@@ -14,3 +14,9 @@ def render(self, context, title):
         template = self.template_name
         context.update(self.get_context_data(title=title))
         return render_to_string(template, context)
+
+    def __copy__(self):
+        return self
+
+    def __deepcopy__(self, memo):
+        return self
```

Comment:
výše používám deepcopy, problém je v tom když někdo v metodě get_fieldsets dá do inline_views už inicializovanou třídu (instanci) pak nechci na takovou třídu provést deepcopy (ani to nejde) takže ta vrátí sama sebe